specs/InvertedIndexSpec.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const InvertedIndex = require('../public/builds/inverted-index').InvertedIndex;
+const InvertedIndex = require('../public/builds/InvertedIndex').InvertedIndex;
 
-const bookGood = require('../booksamples/books.json');
+const books = require('../booksamples/books.json');
 const booksInvalidContent = require('../booksamples/books-invalid-content.json');
 const bookBadJson = require('../booksamples/bad.json');
 const bookEmptyJson = require('../booksamples/book-empty.json');
```

Comment:
A base filename should exactly match the name of its default export. [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)